### ISSUE TITLE
`make PREFIX=/path/to/sl install-oss` broken due to abomonation

### ISSUE THREAD

**Issue by discentem** [facebook/sapling#483]
```bash
warning: /Users/brandon/sapling/configerator/structs/scm/hg/hgclientconf/Cargo.toml: version requirement `0.0.1+unstable` for dependency `fbthrift` includes semver metadata which will be ignored, removing the metadata is recommended to avoid confusion
    Updating git repository `https://github.com/markbt/deltae`
    Updating git repository `https://github.com/vmagro/reqwest`
    Updating crates.io index
    Updating git repository `https://github.com/facebookexperimental/rust-shed.git`
    Updating git repository `https://github.com/facebook/watchman.git`
    Updating git repository `https://github.com/facebook/fb303.git`
    Updating git repository `https://github.com/facebook/fbthrift.git`
error: failed to select a version for `abomonation`.
    ... required by package `dag-types v0.1.0 (/Users/brandon/sapling/eden/scm/lib/dag/dag-types)`
    ... which satisfies path dependency `dag-types` of package `dag v0.1.0 (/Users/brandon/sapling/eden/scm/lib/dag)`
    ... which satisfies path dependency `dag` of package `bindag v0.1.0 (/Users/brandon/sapling/eden/scm/lib/dag/bindag)`
    ... which satisfies path dependency `bindag` of package `dag-benches v0.1.0 (/Users/brandon/sapling/eden/scm/lib/dag/benches)`
versions that meet the requirements `^0.7` are: 0.7.3, 0.7.2, 0.7.1, 0.7.0

the package `dag-types` depends on `abomonation`, with features: `smallvec` but `abomonation` does not have these features.


failed to select a version for `abomonation` which could resolve this conflict
error: compilation of Rust target 'conch_parser' failed
make: *** [oss] Error 1
```

**Comment by vegerot** [comment]
@discentem I fixed this in #478

**Comment by discentem** [comment]
ah thanks @vegerot!
